helm/g2p-sandbox/values.yaml

```diff
@@ -346,4 +497,4 @@ ph-ee-engine:
         enabled: false
       ingress:
         enabled: true
-        hostname: admin-kong.sandbox.mifos.io
+        hostname: admin-kong.sandbox.fynarfin.io
```

Comment:
keep this as mifos.io and change from cookbooks

Reply:
Changed